Sources/VaporToolbox/Build.swift

```diff
@@ -0,0 +1,84 @@
+import Console
+
+public final class Build: Command {
+    public let id = "build"
+
+    public let signature: [Argument] = [
+        Option(name: "run", help: ["Runs the project after building."]),
+        Option(name: "clean", help: ["Cleans the project before building."])
+    ]
+
+    public let help: [String] = [
+        "Compiles the application."
+    ]
+
+    public let console: Console
+
+    public init(console: Console) {
+        self.console = console
+    }
+
+    public func run(arguments: [String]) throws {
+        if arguments.options["clean"]?.bool == true {
+            let clean = Clean(console: console)
+            try clean.run(arguments: arguments)
+        }
+
+        let fetch = Fetch(console: console)
+        try fetch.run(arguments: [])
+
+        var buildFlags: [String] = [
+            "-Xswiftc",
+            "-I/usr/local/include/mysql",
```

Comment:
Not sure if the intention was to leave these, but wanted to mention that these flags are breaking users builds who don't have mysql installed. We should look to circumvent them if possible.